x/incentives/keeper/gauge.go

```diff
@@ -251,14 +280,6 @@ func (k Keeper) CreateGroup(ctx sdk.Context, coins sdk.Coins, numEpochPaidOver u
 		return 0, err
 	}
 
-	// TODO: change CreateGroup method to take in pool IDs
-	// Tracked in issue https://github.com/osmosis-labs/osmosis/issues/6404
-	poolIDs := []uint64{}
-	initialInternalGaugeInfo, err := k.initGaugeInfo(ctx, poolIDs)
-	if err != nil {
-		return 0, err
-	}
-
```

Comment:
Note: I moved this up because it makes sense to check whether the gauge infos can be initialized for the given pool Ids before doing bank transfers.

Not a major change but felt preferable